packages/react-spinner/src/components/Spinner/useSpinnerStyles.ts

```diff
@@ -104,6 +104,9 @@ const useRootStyles = makeStyles({
 const useLoaderStyles = makeStyles({
   // global SVG class
   spinnerSVG: {
+    ':focus-visible': {
+      outlineStyle: 'none',
```

Comment:
Question: should this be a transparent style so that Windows High Contrast Mode can still apply the outline?

cc: @smhigley

Reply:
WHCM still applies the outline, you can test it with input or textarea.